<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UiMain</class>
 <widget class="QWidget" name="UiMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>972</width>
    <height>700</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1080</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>UiMain</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,4">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(85, 170, 255);
border-radius:7px;padding:0px 0px;QWidget { background-color: transparent;}
QWidget * { background-color: transparent; }</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="3,1,0">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3" stretch="2,0,0">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="0,0,0">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="accessibleDescription">
               <string/>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="../../../static/resource.qrc">:/image/drug_select.png</pixmap>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::LinksAccessibleByMouse</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_8">
              <property name="font">
               <font>
                <family>Agency FB</family>
                <pointsize>16</pointsize>
               </font>
              </property>
              <property name="text">
               <string>医院信息管理系统</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>当前用户：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelShowUser">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4" stretch="2,0">
        <item>
         <widget class="QLabel" name="labelTime">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelYearMonthDay">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2" columnminimumwidth="1,1,1,0">
        <item row="0" column="3">
         <widget class="QPushButton" name="btnCloseWidget">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>X</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="btnMaxWidget">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>口</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="btnMinWidget">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="accessibleName">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>-</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="btnQuit">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="accessibleName">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>quit</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <spacer name="verticalSpacer_5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="2">
         <spacer name="verticalSpacer_6">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="3">
         <spacer name="verticalSpacer_7">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widgetManage" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>57</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">
QFrame {
    border: 0px solid black; /* 边框宽度为0，颜色任意 */
	background-color: rgb(85, 255, 255);
	border-style: none; /* 边框样式设置为none */
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="labelLeft">
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
    background-color: transparent; /* 设置背景颜色为透明 */
    border: none; /* 移除边框 */
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>:/Images/left.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnHome">
        <property name="maximumSize">
         <size>
          <width>134</width>
          <height>55</height>
         </size>
        </property>
        <property name="text">
         <string>主页</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSystem">
        <property name="maximumSize">
         <size>
          <width>134</width>
          <height>55</height>
         </size>
        </property>
        <property name="text">
         <string>系统管理</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnOutpatientManage">
        <property name="maximumSize">
         <size>
          <width>134</width>
          <height>55</height>
         </size>
        </property>
        <property name="text">
         <string>门诊管理</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDoctor">
        <property name="maximumSize">
         <size>
          <width>135</width>
          <height>55</height>
         </size>
        </property>
        <property name="text">
         <string>门诊医生</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnToll">
        <property name="maximumSize">
         <size>
          <width>135</width>
          <height>55</height>
         </size>
        </property>
        <property name="text">
         <string>门诊收费</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnInpatientManage">
        <property name="maximumSize">
         <size>
          <width>135</width>
          <height>55</height>
         </size>
        </property>
        <property name="text">
         <string>住院管理</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnPharmacyManage">
        <property name="maximumSize">
         <size>
          <width>135</width>
          <height>55</height>
         </size>
        </property>
        <property name="text">
         <string>药房管理</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDrugManage">
        <property name="maximumSize">
         <size>
          <width>135</width>
          <height>55</height>
         </size>
        </property>
        <property name="text">
         <string>药库管理</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnFinancialManage">
        <property name="maximumSize">
         <size>
          <width>134</width>
          <height>55</height>
         </size>
        </property>
        <property name="text">
         <string>财务管理</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelRight">
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
    background-color: transparent; /* 设置背景颜色为透明 */
    border: none; /* 移除边框 */
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>:/Images/right.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="mainWidget" native="true">
     <property name="styleSheet">
      <string notr="true">
QWidget#mainWidget{
border-image: url(:/image/main.jpg);
}</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../static/resource.qrc"/>
 </resources>
 <connections/>
</ui>
